jina/parsers/orchestrate/pod.py

```diff
@@ -227,3 +233,18 @@ def mixin_pod_runtime_args_parser(arg_group, pod_type='worker'):
         default=None,
         help='If tracing is enabled, this port will be used to configure the metrics exporter agent.',
     )
+
+
+def mixin_hub_pull_options_parser(parser):
+    """Add the arguments for hub pull options to the parser
+    :param parser: the parser configure
+    """
+
+    gp = add_arg_group(parser, title='Pull')
+    gp.add_argument(
+        '--force-update',
+        '--force',
+        action='store_true',
+        default=False,
+        help='If set, always pull the latest Hub Executor bundle even it exists on local',
+    )
```

Comment:
I think we should just use the mixin mixin_hub_pull_options_parser from hubble, shouldn't take time to have hubble sdk release new version

Reply:
I do not think Hubble-sdk should redefine this on their end

Reply:
I do not want to force hubble sdk to change because of this